src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Templating\Tests\Storage\FileStorageTest;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class DownloadBuildCommand extends Command
+{
+    protected static $defaultName = 'sulu:admin:download-build';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    /**
+     * @var string
+     */
+    private $suluVersion;
+
+    /**
+     * @var string
+     */
+    private $remoteRepository;
+
+    /**
+     * @var string
+     */
+    private $remoteArchive;
+
+    const ASSETS_DIR = DIRECTORY_SEPARATOR . 'assets' . DIRECTORY_SEPARATOR . 'admin' . DIRECTORY_SEPARATOR;
+
+    const BUILD_DIR = DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . 'build' . DIRECTORY_SEPARATOR . 'admin';
+
+    const REPOSITORY_NAME = 'skeleton';
+
+    public function __construct(HttpClientInterface $httpClient, string $projectDir, string $suluVersion)
+    {
+        parent::__construct();
+
+        $this->httpClient = $httpClient;
+        $this->projectDir = $projectDir;
+        $this->suluVersion = $suluVersion;
```

Comment:
I would check if suluVersion is a valid sulu version and else ask the developer which version he want to download.

Reply:
I thought about just throwing an exception when it is not a numeric version, because if you are on a develop branch you have to build it on your own anyway (it's not guaranteed for dev branches that the build is up to date).

Reply:
@danrot throwing an exception is ok for me, should just contain a text like: "Looks like you are not a tagged version of sulu, you need to build the admin js yourself"